pkg/blockerslack/reporters/blockers/blockers_reporter.go

```diff
@@ -236,7 +236,7 @@ func triageBug(who string, bugs ...*bugs.Bug) triageResult {
 		r.severityCount[bug.Severity]++
 		r.priorityCount[bug.Priority]++
 
-		if !bug.ReviewedInSprint() {
+		if !bug.ReviewedInSprint() && strings.Contains(bug.TargetRelease, "premerge") {
```

Comment:
is this condition right? should it be `!`strings.Contains....

Reply:
indeed :(

Reply:
fixed